pkg/disttask/framework/scheduler/scheduler.go

```diff
@@ -541,16 +543,6 @@ func (s *BaseScheduler) onNextStage() (err error) {
 		return s.taskMgr.SucceedTask(s.ctx, s.Task.ID)
 	}
 
-	// Adjust the task's concurrency.
-	if s.Task.State == proto.TaskStatePending {
```

Comment:
why delete these?

Reply:
concurrency shouldn't adjust in scheduler